pkgs/applications/science/math/gap/default.nix

```diff
@@ -101,6 +101,9 @@ stdenv.mkDerivation rec {
       excludes = ["tst/testinstall/stringobj.tst"];
       sha256 = "1mz5b4mbw2jdd1ypp5s0dy6pp0jsvwsxr2dm4kbkls20r1r192sc";
     })
+
+    # fixes aarch64 gc crashes: https://github.com/gap-system/gap/pull/3965
+    ./mark-genstackfuncbags-as-noinline.patch
```

Comment:
I think it would be better to use `fetchpatch` to fetch the backport (https://github.com/gap-system/gap/commit/f0a8f49ff8dad0a5fa77253d45457c6f40f96778). That way it would be very explicit which upstream commit the patch corresponds to. This is not a blocker for the merge, just something I wanted to mention.

Reply:
I appreciate the suggestion! I tried doing this originally, but there was a previous upstream change at the exact same line (adding the `static` keyword to the method definition) so the patch didn't apply.

Reply:
Ah, you are right. I missed the `static` keyword. Good work :)